UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Fugitive.cs

```diff
@@ -53,7 +53,7 @@ public override GameObject ServerSpawn(PlayerSpawnRequest spawnRequest)
 				" I need to move in the shadows and keep out of sight," +
 				" I'm not going back.");
 
-			_ = StationWarning(newPlayer.ExpensiveName());
+			_ = StationWarning(newPlayer.Player().Name);
```

Comment:
this one was supposed to be ic name

Reply:
So it's fine if it shows up as `Unknown`?